### PR TITLE
Stats command

### PR DESCRIPTION
**Pull request by Ma-wa-re**
Added a stats command that has a link to this repo.
Shows the uptime of the selfbot and the number of guilds, channels, and users it can see.
It also shows Repo info and System info.

Completes #12

### CODE REVIEW

File: mawabot/cogs/info.py [hunk 1]
```diff
@@ -13,15 +13,22 @@
...
+import psutil
+
+from mawabot import __version__ as version
 
 __all__ = [
     'setup',
 ]
 
 EMOJI_REGEX = re.compile(r'<:([A-Za-z~\-0-9]+):([0-9]+)>')
+GIT_REPO_CONTRIBUTORS = 'https://api.github.com/repos/strinking/mawabot/stats/contributors'
```
Comment (emmiegit) [inline mawabot/cogs/info.py]: Maybe make a `REPO` variable and then add it?

```py
REPO = 'strinking/mawabot'
GITHUB_CONTRIBUTORS = f'https://api.github.com/repos/{REPO}/stats/contributors'
```

File: mawabot/cogs/info.py [hunk 2]
```diff
@@ -75,6 +82,21 @@ def _get_members(self, guild, names):
...
+        while True:
+            async with aiohttp.ClientSession() as session:
+                async with session.get(GIT_REPO_CONTRIBUTORS) as resp:
+                    status = resp.status
+                    data = await resp.json()
+                
+            if status == 200:
+                return data
+            elif status == 202:
+                await asyncio.sleep(10)
```
Comment (emmiegit) [inline mawabot/cogs/info.py]: What's this for? It'll just implicitly return `None` afterwords, no?

File: mawabot/cogs/info.py [hunk 3]
```diff
@@ -161,6 +183,43 @@ def _get_members(self, guild, names):
         uptime = str(self.bot.uptime).split('.')[0]
         await ctx.message.edit(content=f'`{uptime}`')
 
+    @commands.command()
+    async def stats(self, ctx):
+        ''' Gets bot stats '''
+
+        title = 'mawabot'
+        url = 'https://github.com/strinking/mawabot'
```
Comment (emmiegit) [inline mawabot/cogs/info.py]: This should be coupled with `REPO` from above.

```python
REPO = 'strinking/mawabot'
GITHUB_URL = 'https://github.com/{REPO}'
```

File: mawabot/cogs/info.py [hunk 4]
```diff
@@ -161,6 +183,43 @@ def _get_members(self, guild, names):
...
+                f'Guilds: `{len(self.bot.guilds)}`',
+                f'Channels: `{channels}`',
+                f'Users: `{len(self.bot.users)}`',]
+
+        embed = discord.Embed(title=title, url=url, description='\n'.join(desc))
+        git = []
+        contributors = await self.get_git_contributors()
+
+        if contributors:
+            contributors = sorted(contributors, key=lambda x: x.get('total'), reverse=True)
```
Comment (emmiegit) [inline mawabot/cogs/info.py:269]: 👍

File: mawabot/cogs/info.py (hunk 2 above)
Comment (jchristgit) [inline mawabot/cogs/info.py]: Isn't 10 seconds a bit too long?

File: mawabot/cogs/info.py [hunk 5]
```diff
@@ -161,6 +183,43 @@ def _get_members(self, guild, names):
...
+                git.append(f'[{author["login"]}]({author["html_url"]}) ({user["total"]})')
+
+        embed.add_field(name='Repo Info', value=f'Version: `{version}`\n{", ".join(git)}')
+
+        # Get system info
+        pid = os.getpid()
+        py = psutil.Process(pid)
+
+        cpu = py.cpu_percent()
+        mem = py.memory_info()[0]/1000/1000
```
Comment (emmiegit) [inline mawabot/cogs/info.py]: I think `py.memory_info()[0] / MB` would be more clear.

Also imo we should use MiB not MB.
```python
MB = 1000 *1000
MiB = 1024 * 1024
```

File: mawabot/cogs/info.py (hunk 2 above)
Comment (Ma-wa-re) [reply mawabot/cogs/info.py]: Most likely but it makes sure. I did that for testing and forgot to change it

File: mawabot/cogs/info.py [hunk 6]
```diff
@@ -13,15 +13,22 @@
 ''' Has several commands that get guild information '''
 import re
 import unicodedata
+import aiohttp
+import asyncio
```
Comment (jchristgit) [inline mawabot/cogs/info.py]: Sort imports! `isort` is a nice tool for this.

**Review by emmiegit — COMMENTED** [review]
Looks good. Just some minor things.

**Review by emmiegit — APPROVED** [review]

**Pull request merged**